sql/database.sql

```diff
@@ -1178,6 +1178,7 @@ CREATE TABLE `documents` (
   `audit_master_id` int(11) default NULL,
   `documentationOf` varchar(255) DEFAULT NULL,
   `encrypted` TINYINT(4) NOT NULL DEFAULT '0' COMMENT '0->No,1->Yes',
+  `document_data` MEDIUMTEXT NULL DEFAULT NULL
```

Comment:
This is `MEDIUMTEXT` and I am requesting that you remove extra burden of `NULL DEFAULT NULL`. The pressure is immense for us `TEXT` folks and removing this would immensely simplify my purpose.
Sincerely,
Your fav `TEXT`,
`MEDIUMTEXT`